workers/unity/Packages/com.improbable.gdk.deploymentlauncher/Commands/Assembly.cs

```diff
@@ -19,7 +19,8 @@ internal static class Assembly
                 config.AssemblyName,
                 "--project_name",
                 config.ProjectName,
-                "--json_output"
+                "--json_output",
+                "--enable_pre_upload_check=false"
```

Comment:
add a comma at the end so that you don't need to change this line if we ever need to update this